seed/static/seed/partials/inventory_detail_sensors.html

```diff
@@ -33,10 +33,26 @@ <h2>
 </div>
 <div class="section_content_container" ng-cloak style="padding-left:15px; padding-right:15px;">
 
-    <div id="sensor-info-container">
-        <div class="btn-group" style="padding-top: 15px;">
-            <button type="button" ng-click="open_sensor_upload_modal()" class="btn btn-primary" translate>UPLOAD_SENSOR_BUTTON</button>
+    <div>
+        <div class="btn-group" style="padding-top: 15px; display: flex; gap: 15px;">
+            <button type="button" style="border-radius: 4px;" ng-click="open_data_logger_upload_modal()" class="btn btn-primary" translate>UPLOAD_DATA_LOGGER_BUTTON</button>
```

Comment:
I'm thinking we should have "Add Data Logger" which is just a modal form to create the data logger. Then use your already created "Upload Sensor Readings" to the move on to that step.

Also when giving a random CSV file, the `SENSOR_CONTENTS_TITLE` doesn't resolve (see image)

<img width="322" alt="image" src="https://user-images.githubusercontent.com/1907354/159022018-1adc3433-c98b-4dc6-a356-30736d548868.png">